tests/shell/check_kdb_internal_check.sh

```diff
@@ -56,6 +56,9 @@ do
 	ASAN='@ENABLE_ASAN@'
 	if [ "$ASAN" = 'ON' ]; then
 		case "$PLUGIN" in
+		'augeas') # Reference: https://travis-ci.org/sanssecours/elektra/jobs/418524229
```

Comment:
Also here it would be nice to automatically exclude plugins based on their contract...

Reply:
Thanks for the advice. I changed the behavior accordingly in commit 4d1a6dcf.